lldb/include/lldb/Target/Process.h

```diff
@@ -2800,6 +2809,10 @@ void PruneThreadPlans();
   virtual size_t DoReadMemory(lldb::addr_t vm_addr, void *buf, size_t size,
                               Status &error) = 0;
 
+  void DoFindInMemory(lldb::addr_t start_addr, lldb::addr_t end_addr,
+                      const uint8_t *buf, size_t size, AddressRanges &matches,
+                      size_t alignment, size_t max_matches);
+
```

Comment:
We might want this to be virtual to allow core files to do this quicker than having to read all memory into a buffer.